api/edge_api/identities/serializers.py

```diff
@@ -142,6 +153,24 @@ def save(self, **kwargs):
         return self.instance
 
 
+class EdgeIdentityIdentifierSerializer(serializers.Serializer):
+    identifier = serializers.CharField(required=True, max_length=2000)
+
+
+# NOTE: This is only used for generating swagger docs
+class EdgeIdentityWithIdentifierFeatureStateRequestBody(
+    EdgeIdentityFeatureStateSerializer, EdgeIdentityIdentifierSerializer
+):
+    pass
+
+
+# NOTE: This is only used for generating swagger docs
+class EdgeIdentityWithIdentifierFeatureStateDeleteRequestBody(
+    EdgeIdentityIdentifierSerializer
+):
+    feature = EdgeFeatureField(help_text="ID or name of the feature")
```

Comment:
Minor point but can we default the `help_text` argument in the `__init__` method of the `EdgeFeatureField` class?